_includes/docker_platform_matrix.md

```diff
@@ -1,16 +1,16 @@
-| [Fedora]({{ install-prefix }}/docker-ce/fedora.md)                         | [{{ green-check }}]({{ install-prefix }}/docker-ce/fedora.md) |                                                               |                                                               |                                                                |                                                               |
-| [Microsoft Windows Server 2016](/engine/installation/windows/docker-ee.md) |                                                               |                                                               |                                                               | [{{ green-check }}](/engine/installation/windows/docker-ee.md) |                                                               |
-| [Oracle Linux]({{ install-prefix }}/docker-ee/oracle.md)                   |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/oracle.md)  |                                                               |
-| [Red Hat Enterprise Linux]({{ install-prefix }}/docker-ee/rhel.md)         |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/rhel.md)    | [{{ green-check }}]({{ install-prefix }}/docker-ee/rhel.md)   |
-| [SUSE Linux Enterprise Server]({{ install-prefix }}/docker-ee/suse.md)     |                                                               |                                                               |                                                               | [{{ green-check }}]({{ install-prefix }}/docker-ee/suse.md)    | [{{ green-check }}]({{ install-prefix }}/docker-ee/suse.md)   |
-| [Ubuntu]({{ install-prefix }}/ubuntu.md)                                   | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ce/ubuntu.md) | [{{ green-check }}]({{ install-prefix }}/docker-ee/ubuntu.md)  | [{{ green-check }}]({{ install-prefix }}/docker-ee/ubuntu.md) |
+| Platform                                                                   | Docker CE x86_64                                              | Docker CE ARM                                                 | Docker CE IBM POWER (ppc64le)                                 | Docker CE IBM Z (s390x)                                       | Docker EE x86_64                                               | Docker EE IBM Z (s390x)                                       |
```

Comment:
I was just basing this off of the s390x one, but because this is going to get cluttered fast with ee ppc64le, and arm64, do you think we should change the formatting to `Docker CE/EE $ARCH` like arm and x86 does?